ScoutSuite/output/data/html/partials/azure/services.appgateway.app_gateways.html

```diff
@@ -0,0 +1,25 @@
+<!-- Application Gateway partial -->
+<script id="services.appgateway.app_gateways.partial" type="text/x-handlebars-template">
+    <div class="list-group-item active">
+        <h4 class="list-group-item-heading">{{name}}</h4>
+    </div>
+
+    <div class="list-group-item">
+        <h4 class="list-group-item-heading">Information</h4>
+        <div class="list-group-item-text item-margin">Web Application Firewall: <span id="appgateway.app_gateways.{{@key}}.web_app_firewall_enabled">{{ convert_bool_to_enabled web_app_firewall_enabled }}</span></div>
+    </div>
+
+</script>
+
+<script>
+    Handlebars.registerPartial("services.appgateway.app_gateways", $("#services\\.appgateway\\.app_gateways\\.partial").html());
+</script>
+
+<!-- Single Application Gateway template -->
+<script id="single_appgateway_app_gateways-template" type="text/x-handlebars-template">
+    <div style="text-align: right; padding-right: 10px; text-weight: bold;"><a href="javascript:hidePopup()">X</a></div>
+    {{> services.appgateway.app_gateways}}
```

Comment:
Please prefer using the modal partial instead of this implementation. The javascript:hidePopup does not exist anymore.  See [this](https://github.com/nccgroup/ScoutSuite/pull/132/files#diff-795b3bcacbbb37ba1939693c79cb20a7) example.

Reply:
Done, sorry about that. :x